config/locales/liquid_tags/en.yml

```diff
@@ -99,7 +99,9 @@ en:
       invalid_url: Invalid Twitter Timeline URL
     unified_embed:
       tag:
-        invalid_url: Embed URL not valid
+        invalid_url: URL provided may have a typo or error; please check and try again
```

Comment:
In testing I wasn't able to trigger this one 😄  I always got `not_found` - not sure what I'm missing in the test steps

Reply:
This error should get triggered when a url doesn't exist. For example if one was to do `{% embed https://thisdomainreallydoesnotexist.wtf %} it should trigger that error.

Reply:
Can confirm that works 🙌  

I tried typoing some URLs that are normally valid (e.g. changing a single letter in a loom or github embed link), in which case I get something less readable, but I'm guessing this is just because it's genuinely triggering some more detailed/specific error - e.g.

![error: SSL_connect SYSCALL returned=5 errno=0 state=SSLv3/TLS write client hello](https://user-images.githubusercontent.com/20773163/157693131-41ea3925-29a4-43e2-8f1c-674bd30b69a6.png)

Reply:
Ok so @aitchiss I was not able to reproduce your experience of that wonky error message. 🤷🏾‍♀️ 

I recorded a Loom of myself triggering the "URL not found" and "URL may have typo" error messages, and things seem to be working okay:

https://www.loom.com/share/a19a44700b764d438eda13d5760b69e7

I'll wait for feedback from @dscottS3 @aitchiss before proceeding to merge this PR 👍🏾